src/client/components/header/search/Search.js

```diff
@@ -41,11 +53,18 @@ class Search extends React.Component {
       'active': this.state.searchByGengre.status,
     })
 
+    let searchClass = cn({
```

Comment:
it always return true and always return `search-input-button`, If we need to apply some style which depend on some condition we will use such construction. In other case your can assign this class to `toggleClassName`

Reply:
Corrected)